rmgpy/rmg/model.py

```diff
@@ -675,31 +675,6 @@ def enlarge(self, new_object=None, react_edge=False,
         if not np.isinf(self.thermo_tol_keep_spc_in_edge) and self.new_species_list != []:
             self.thermo_filter_species(self.new_species_list)
 
-        # Generate kinetics of new reactions
-        if self.new_reaction_list:
-            logging.info('Generating kinetics for new reactions...')
```

Comment:
Not sure how important it is, but maybe we could move this logging info line somewhere for the user